pillar.example

```diff
@@ -0,0 +1,20 @@
+netweaver:
+  version: 7.5
+  sid: HA1
+  virtual_addresses:
+    192.168.201.111: hacert01
+    192.168.201.112: hacert02
+    192.168.201.115: sapha1as
+    192.168.201.116: sapha1er
+  sapcd_media: your_nfs_share_SWPM_folder
```

Comment:
I think `sapcd` is a old reference of a real SAP CD that was distributed. I would change it to something like `swpm_media`